script/Ciel/Day 8/08_01_CIEL08_1E.txt

```diff
@@ -102,7 +102,7 @@ As a reaction to the changes their bodies undergo,
 -- Page 137, Offset 18622.
 -- だって一度も着れなかったし。お姫さまみたいなドレスでね、着こなすには勇気が必要だったの。そんな勇気が出る前に大人になっちゃったけど」
 I never even got the chance to wear them!
- It takes a lot of courage to wear a princess-y dress,
+ It takes a lot of courage to wear a princessy dress,
```

Comment:
kinda prefer princess-y, though this line is probably the most contentious in the whole script at this point